server/routes/auth/providers/email.test.ts

```diff
@@ -41,8 +42,8 @@ describe("email", () => {
   });
 
   it("should respond with redirect location when user is SSO enabled on another subdomain", async () => {
-    env.URL = "http://localoutline.com";
-    env.SUBDOMAINS_ENABLED = true;
+    env.URL = sharedEnv.URL = "http://localoutline.com";
+    env.SUBDOMAINS_ENABLED = sharedEnv.SUBDOMAINS_ENABLED = true;
```

Comment:
Because the new env interface decouples `process.env` from `server/env` it also means any setting of values doesn't propagate.

Because the dependency injection for `shared/env` comes directly from `process.env` we need to set it explicitly as well.

I hate this!

Reply:
Not ideal, but I can deal.